application/language/russian/index.html

```diff
@@ -1,10 +1,11 @@
+<!DOCTYPE html>
 <html>
 <head>
-	<title>403 Запрещено</title>
+	<title>403 Forbidden</title>
```

Comment:
Probably you wouldn't want to change this.

Reply:
This is non sense to keep these translated, because this file is loaded only if one tries to reach that directory directly. It has nothing to do with language choicee in kalkun & translation of kalkun. The only translation file used is kalkun_lang.php

Reply:
Well true, but say tomorrow we want to dynamically use the translated file for all 404 pages. The idea then would be to create a templatised 4XX pages. At that time we would need these translations, which we could just resuse from these files. If we remove these translations now, then we will have to put effort again for translations, hence IMO its better to keep these as it is, and change this later.

Reply:
Then I prefer to put these few cases in the language file and leave index.html as is.

Reply:
Done. Created a new commit with the translated labels in the translation files for future use.